src/app/shared/components/child-card/child-card.component.ts

```diff
@@ -1,27 +1,32 @@
+// import { Application } from 'src/app/shared/models/application.model';
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { Store } from '@ngxs/store';
+import { Select, Store } from '@ngxs/store';
+import { Observable } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { CabinetDataComponent } from 'src/app/shell/personal-cabinet/cabinet-data/cabinet-data.component';
+import { Role } from '../../enum/role';
+import { Application } from '../../models/application.model';
 import { Child } from '../../models/child.model';
 import { Workshop } from '../../models/workshop.model';
 import { DeleteChildById } from '../../store/user.actions';
+import { UserState } from '../../store/user.state';
 
```

Comment:
delete please imports that we don't need
![image](https://user-images.githubusercontent.com/27493959/137163236-5fa0828b-60cc-4151-90cb-9d7b5682320a.png)